build.gradle.kts

```diff
@@ -72,15 +72,18 @@ plugins {
 }
 
 buildscript {
-    repositories {
-        gradlePluginPortal()
-        google()
-    }
-    dependencies {
-        classpath("gradle.plugin.com.github.johnrengelman:shadow:8.0.0")
+  repositories {
+    maven {
+      url = uri("https://plugins.gradle.org/m2/")
     }
+  }
+  dependencies {
+    classpath("com.github.johnrengelman:shadow:8.1.1")
+  }
 }
 
+apply(plugin = "com.github.johnrengelman.shadow")
```

Comment:
Do you need to apply this? If it was attached to build script classpath its already there for building.

Reply:
again, this is how it was at https://plugins.gradle.org/plugin/com.github.johnrengelman.shadow

Reply:
Looks like that says two different methods though and you have both, so just a bit lost.